packages/components/Switch/jest.config.js

```diff
@@ -0,0 +1,2 @@
+const { configureReactNativeJest } = require('@fluentui-react-native/scripts');
+module.exports = configureReactNativeJest('android');
```

Comment:
I'm a little unsure why a couple files got added instead of just renamed -I think there is a packages/experimental/Switch/jest.config.js file, and a packages/experimental/Switch/src/switch.types.ts file, so these should be registered as "file renamed without changes" instead of as a new file.